src/App.js

```diff
@@ -1,14 +1,17 @@
-import { Switch, Route } from 'react-router-dom';
+import { Switch, Route, Redirect } from 'react-router-dom';
 
 import ContactPage from './pages/Contact';
 import TeamPage from './pages/Team';
+import Menu from './components/Menu';
 
 function App() {
   return (
     <div className="App">
+      <Menu />
       <Switch>
+        // Might make homepage later— for now, root directs to team page.
         <Route path="/" exact>
-          <TeamPage />
+          <Redirect to="/team" />
```

Comment:
I like how you redirect from "/" to "/team". I just learned that there is a way to make these three lines into one line just to make the code look simpler. You can write: `<Redirect exact from="/" to="/team" />` so that you do not need to put the Redirect statement inside of a Route tag.